diff --git a/data/org.gnome.dspy.appdata.xml.in.in b/data/org.gnome.dspy.appdata.xml.in.in
index 3745ab9..c76512e 100644
--- a/data/org.gnome.dspy.appdata.xml.in.in
+++ b/data/org.gnome.dspy.appdata.xml.in.in
@@ -19,7 +19,7 @@
   <screenshots>
     <screenshot type="default">
       <caption>D-Spy Interface</caption>
-      <image width="2104" height="1598">https://gitlab.gnome.org/chergert/d-spy/-/raw/main/data/dspy-1.png</image>
+      <image>https://gitlab.gnome.org/GNOME/d-spy/-/raw/main/data/dspy-1.png</image>
     </screenshot>
   </screenshots>
